package.json

```diff
@@ -38,17 +41,20 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@types/cookie-parser": "~1.4.6",
     "@types/express": "^4.17.13",
     "@types/jest": "^27.4.1",
     "@types/node": "^18.11.18",
     "@types/nunjucks": "^3.2.1",
+    "@types/sinon": "~7.5.2",
     "@types/supertest": "^2.0.11",
     "@typescript-eslint/eslint-plugin": "^5.16.0",
     "@typescript-eslint/parser": "^5.16.0",
     "copyfiles": "^2.4.1",
     "eslint": "^7.32.0",
     "jest": "^27.5.1",
     "nodemon": "^3.0.1",
+    "sinon": "~9.0.1",
```

Comment:
Do you need sinon? I don't see it being used in the PR

Reply:
thanks @JoelClemence  I have modified it.